FlorenceBML/BMLacq38.xml

```diff
@@ -34,6 +35,376 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                        3) Vari opuscoletti in cronologia, tra cui (f. 137r) la lista dei Re di Etiopia che giunge sino agli inizi del secolo XVI (età del Codice).
+                                            Questa lista servì di base al «Catalogo dei Re» di Vittorio Mariano, etiopista italiano del secolo XVI che - quasi certamente - adoperò 
+                                            il presente Codice.
+                                       4) Il rituale (mutilo) della Comunione (secondo la liturgia etiopica) 
+                                        <persName ref="PRS3076Cerulli">E. Cerulli</persName>."
+                                                </foreign></desc>
+                                </item>
+                                <item xml:id="e14">
+                                    <desc type="StampExlibris">Paper leaf that has been folded and written on both sides with pen by D. Valeriani in the nineteenth 
+                                        century to Signore Formigli being attached to the inner rear board and containing a transcription and translation of the first sentences 
+                                        of the codex, and an arbitrary description of its content:
+                                        <foreign xml:lang="it">
+                                                "Amico
+                                            Il noto Codice Etiopico incomincio cosi.<gap reason="ellipsis"/>
+                                            pasticcio informe <gap reason="ellipsis"/>
+                                            le tante coglionerie, ond'è ripiena<gap reason="ellipsis"/>
+                                            libro ... astrologico-mistico-giudiziario<gap reason="ellipsis"/>
+                                            in puro Etiopico, ma in Etiopico mischiato di Amharico, che è la Lingua volgare degli Abissini.<gap reason="ellipsis"/>"
+                                            </foreign>
+                                                </desc>
+                                </item>
+                            </list>
+                        </additions>
+                        <bindingDesc>
+                            <binding contemporary="false" xml:id="binding">
+                                <decoNote xml:id="b1" type="Boards">Two wooden boards.</decoNote> 
+                                <decoNote xml:id="b2" type="Cover">European wooden binding, with leather covering the spine and half of the outer plates.</decoNote> 
+                                <decoNote xml:id="b3" type="bindingMaterial">
+                                    <material key="wood"/>
+                                    <material key="leather"/>
+                                </decoNote>
+                            </binding>
+                        </bindingDesc>
+                    </physDesc>
                     <history>
```

Comment:
I think `<origin>` `<origDate>` should be added, as the ms is possibly datable to the time of Lebna Dengel or has him at least as terminus post quem
                     
```
<origDate notBefore="1497" notAfter="1540"/>
                     <note>The chronological text goes up to <persName ref="PRS6229LebnaDe"> which is also in agreement with the palaeography.</note>
```

Reply:
Done. Can we give two origDate? The one with the time span, that you proposed and additionally a precise dating to 1530/1531 with medium certainty?

Reply:
I mean, even if the date is correct and precise (what I am convinced, now), we are still not sure, whether this date was added later than the main part.

Reply:
Perhaps like that:
```
<origin>
<origDate notBefore="1508" notAfter="1540"/>
<note>The chronological text goes up to <persName ref="PRS6229LebnaDe"/> which is also in agreement with the palaeography.</note>
<origDate notBefore="1530" notAfter="1531" cert="medium"/>
 <note>According to a note on <locus target="143r"/> the manuscript was produced in the 23rd year of <persName ref="PRS6229LebnaDe"/>
 and the 183rd year of Mercy, that correspond with the year 1530 to 1531. Since this note was written most likely by another hand on a bifolium,
 that was probably attached later, it is not certain, whether it is precisely correct.</note>
                        </origin>
```

Reply:
I do not think we can have 2 origDates, I would remove the first one

Reply:
(Even if the additio was added later we still have Lebna Dengel as terminus post quem and we have the note also as terminus ante quem, which still brings us to the same date)

Reply:
Yes, this is alright and brings us to the safe frame of 1503 to 1540. Unless the 3 in 23 and in 183 was erased. Probably the one who wrote the end of the text, reconsidered the date and set a new date to 20 / 180, what is 1527. We cannot know, what is the precise date, in fact. This is why I would give the certainty medium, but not high, if two origDate are useful.